app/routes.py

```diff
@@ -1,2 +1,60 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, request, make_response, abort
+from app import db
+from app.models.planet import Planet
+from .helper import validate_planet
+solar_bp = Blueprint("planets", __name__, url_prefix="/planets")
 
+# GET and POST planets
+@solar_bp.route("", methods=["POST", "GET"])
```

Comment:
It's good practice to keep in mind SRP - single responsibility principle for methods here. 

Each method should handle its own route so you can split the POST and the GET requests into 2 separate methods. Maybe one called create_planet() for POST and get_planets() for GET